experimental/packages/otlp-exporter-base/src/platform/browser/util.ts

```diff
@@ -54,14 +58,17 @@ export function sendWithBeacon(
  * @param body
  * @param url
  * @param headers
+ * @param exporterTimeout
  * @param onSuccess
  * @param onError
+ * @param compressionAlgorithm
  */
 export function sendWithXhr(
```

Comment:
While I'm fine with the breaking change (as we are calling it out as such), I'm wondering if it might be better to introduce an `options` object which wraps all/some of the additional arguments so that if/when we need / want to add some additional parameters in the future this function won't need any breaking changes.

eg.
```ts
sendWithXhr(body, url, {
    headers: Record<string, string>,
    timeout: number,
    compressionAlgorithm: CompressionAlgorithm
},
onSuccess, onError)
```

as an example. I'm fine either way as this is really internal at this point, just thinking about moving forward etc.